src/platform/testing/unit/helpers.js

```diff
@@ -136,7 +144,11 @@ export function setFetchBlobFailure(stub, error) {
  * Resets the fetch mock set with mockFetch
  */
 export function resetFetch() {
-  global.fetch.reset();
+  // To prevent a really unlikely edge case where resetFetch() is called before
+  // mockFetch()
+  if (oldFetch) {
```

Comment:
In order for `msw` to work, we need to fully restore the global fetch function back to its original form.